components/ui/header.tsx

```diff
@@ -1,7 +1,22 @@
 import React from "react";
-
+import Button from "@/components/ui/button";
+import logo from "./assets/Logo.svg"
 function Header() {
-  return <div>header</div>;
+    return (
+      <div className="w-full max-w-5xl p-8 flex justify-between items-center font-normal text-sm mx-auto fixed left-1/2 md:translate-x-[-50%]">
+          <img alt="logo" src={logo.src} className="w-7 h-7 cursor-pointer"/>
+          <div className="flex justify-center items-center gap-7">
+              <div className="cursor-pointer">Jobs</div>
```

Comment:
Here, we will use the `<Link></Link>` component from next.js that will route to our pages. The docs for this feature you can check [docs]( https://nextjs.org/docs/app/api-reference/components/link)